### ISSUE TITLE
powroupi quitting

### ISSUE THREAD

**Issue by Hogarth-MMD** [powroupi/blender_mmd_tools#187]
12 days ago, @powroupi wrote:

"I think the only thing left of my plan is to make mmd_tools work on Blender 2.8, no more plans to update anything anymore. So, it will be the final version of my fork if everything works fine on Blender 2.8. Thanks everyone who helps me to improve this addon, feel free to fork & continue it if you're interested. Thank you. "

I cannot believe that no one replied to this!!!!???? 
People should be responding to this with an outpouring of love and appreciation for @powroupi  and encouragement for him to continue.  

@powroupi has just been an awesome developer of mmd_tools. Awesome both in programming cleverness and in the generous giving of his time. Whenever someone posts a new issue, @powroupi has always been there,, in his beautiful @powroupi style, saying, "Thank you for reporting this issue.", and providing beautiful, generous help to the person who reported the issue.

**Comment by aecii** [comment]
Very thankful and happy to see all the updates. As a Linux user, I dont get to use MMD as much as I would like to, but seeing it work in B3d (2.79) being able to import and study some models has really helped in learning more about 3d arts and animation. Its been a great vehicle for holding my interest and attention while sparking more curiosity. Thank you @powroupi for all the effort you have put into this project. It is very appreciated. I do hope this project would continue well into b3d 2.8+ but time will tell. Thank you @powroupi  for carrying this project further.

**Comment by nathanvasil** [comment]
Yes, powroupi has been amazing.  I cannot express my gratitude for MMD Tools-- without it, I never would have learned Blender, and I doubt I would have gotten very far in 3D modelling using only PMXE.

**Comment by Hogarth-MMD** [comment]
There are many remarkable programming achievements in mmd_tools. mmd_tools is almost like a superior version of Blender. In Blender without mmd_tools, you don't find any physics rigging of characters, or any dual/bilingual naming of anything, or any translation from one language to another, or any UV morphs, or any bone pose morphs, or group morphs. You don't find these features in other Blender add-ons. But mmd_tools has all of these features, thanks to the programming excellence and beautiful, loving care of sugiany, powroupi and a few other people.

mmd_tools is superior to commodities, such as oil. There can be oil shortages and countries fighting over oil. But there is never any shortage of mmd-tools. There is always an endless, beautiful, free supply of mmd_tools for everyone in the world. :smile:

**Comment by powroupi** [comment]
Thank you for the compliments. :kissing_heart:

Well, I think I've done every feature that I want, and I'm afraid I will not have enough spare time to maintain it (maybe waiting for my vacations or retiring), so I think it will be the final version of my fork when everything works fine on Blender 2.8, I'll currently focus on bug fix and minor UI tweaks. :smile:

**Comment by ptrthomas** [comment]
thanks @powroupi I'm a huge fan - and it is a relief actually to know that you will still be observing and perhaps doing a bug fix or two.

I've mentioned this before - mmd_tools has been a source of amazing fun for me, what started as an interest in vocaloid snowballed into me buying books on blender and then getting interested in unreal engine. 

the fact that I can do all of this on a Mac without using Windows at all - is wonderful. and I am proud of the couple of small pull requests I made for mmd_tools - especially the english translation - where I learnt how to use JavaScript in Google Sheets as a bonus :)

P.S. I have mentioned a thanks 🙏 to powroupi in this video at the start: https://www.youtube.com/watch?v=fbbuS7TqiX0

**Comment by Hogarth-MMD** [comment]
mmd_tools is an unfinished masterpiece (like a Mona Lisa without her weird smile). Implementation of material morph slider controls. Rendering of MMD materials. Fixing the 2 adding physics bugs that I mentioned earlier. The completion of compatibility with Blender 2.8. Isn't that all that NEEDS to be done with it now? Do those tasks require too much time?

**Comment by Hogarth-MMD** [comment]
If and when the day arrives that @powroupi has completely stopped making new commits for mmd_tools, I think that I would want to combine mmd_tools, mmd_tools helper, and translations.csv into a single add-on, for the convenience of mmd_tools users. The only problem that I can see with that, is that mmd_tools helper is not updated for Blender 2.8.

**Comment by Hogarth-MMD** [comment]
Sometimes when you don't have enough time for a large task, it's possible to analyze it into a list of smaller tasks, and then do the task slowly, one small task at a time. I'm not really sure if the finishing up of mmd_tools could be done that way. :thinking:

**Comment by gitspyder** [comment]
I'm a huge fan of the tool and all the updates made to it.  Thank you very much for picking up the torch and keeping things improving for as long as you have, especially with the Blender 2.8 update!  You are a titan among men sir!
